packages/app/src/specs/banners/ConnectProjectBanner.vue

```diff
@@ -14,75 +13,43 @@
       medium: 'Specs Create Project Banner',
       cohort: cohortOption.cohort
     }"
-    @update:model-value="value => emit('update:modelValue', value)"
   >
     <p class="mb-24px">
-      {{ cohortOption.value }}
+      <!--
+        unref allows plain object to be used in testing
+        while a ref is provided in actual usage
+      -->
+      {{ unref(cohortOption)?.value }}
```

Comment:
This is a bit of a code-smell, I would expect the test to be more verbose (to accomodate ref vs plain object) rather than leak into app code